CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,92 @@
-﻿
+﻿using System;
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int Addnums(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = i + sum;
+
+            }
+            if (i == 0)
```

Comment:
If i is equal to zero, then return 0. Otherwise, return i?  How about just `return i;` regardless of whether it is zero